### PR TITLE
fix(lang): change many english expression, and fix label on filter list

### PR DESCRIPTION
**Pull request by Debaerdm**
# Pull Request Template

## PR Description

Change many english expression, and fix label on filter list

### CODE REVIEW

File: public/assets/langs/en.json [hunk 1]
```diff
@@ -45,18 +45,18 @@
 	"Conflicts": "Conflicts",
 	"NoRepository": "No repository for this project.",
 	"Back": "Back",
-	"ProjectsOrganizations": "Select the organizations for which you want to see projects.",
-	"OrganizationsEmpty": "There is no valid organization on your account.",
+	"ProjectsOrganizations": "Select the organizations to see their projects",
```
Comment (triathlete23) [inline public/assets/langs/en.json:48]: Select an organization to see projects

File: public/assets/langs/en.json [hunk 2]
```diff
@@ -45,18 +45,18 @@
...
+	"LaunchStartup": "Launch on startup",
+	"ConfigTheme": "Pick your theme",
 	"Version": "Version",
 	"SignOut": "Sign out",
-	"Theme": "Theme",
+	"Theme": "Themes",
 	"Organizations": "Organizations",
-	"StartUp": "Start-up",
-	"Refreshment": "Refreshment",
+	"StartUp": "Auto launch",
```
Comment (triathlete23) [inline public/assets/langs/en.json:58]: Auto**-**launch

File: public/assets/langs/en.json (hunk 1 above)
Comment (triathlete23) [inline public/assets/langs/en.json:48]: Select organization(s) to see projects

**Review by itupix — APPROVED** [review]

File: public/assets/langs/en.json (hunk 1 above)
Comment (Debaerdm) [reply public/assets/langs/en.json:48]: Hey thanks, i'm going to fix it

**Pull request merged**